vpn/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/AppTPRemoveFeatureConfirmationDialog.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2019 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.mobile.android.vpn.ui.tracker_activity
+
+import android.app.Dialog
+import android.os.Bundle
+import android.widget.Button
+import androidx.fragment.app.DialogFragment
+import com.duckduckgo.mobile.android.vpn.R
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+class AppTPRemoveFeatureConfirmationDialog private constructor(private val listener: Listener) : DialogFragment() {
```

Comment:
nit: I think we discussed to not use `AppTP` prefix anymore in favor of `Vpn` which reads better in code.
In that spirit, can we rename this to `VpnRemoveFeatureConfirmationDialog`